django_fastdev/apps.py

```diff
@@ -39,16 +42,43 @@ def ignore_template_errors():
         _local.ignore_errors = False
 
 
+def get_gitignore_path():
+    try:
+        path = settings.BASE_DIR
+    except ImportError:
+        path = settings.ROOT_DIR
+    if os.path.isfile(os.path.join(path,".gitignore")):
```

Comment:
since you're checking for `ROOT_DIR` something that cookiecutter uses, might be smarter to use `pathlib` instead, as `ROOT_DIR` is `Path` object itself iirc